test/profile/test_validation_intervals.py

```diff
@@ -0,0 +1,63 @@
+"""
+This is for testing profile based validation intervals.
+"""
+
+import numpy as np
+import unittest
```

Comment:
Haha, my policy would always be to use pytest instead of the class-syntax, which I find hard to read. Would be possible here for sure, but all would be fine for me.